src/api/cpp/backend/backend_aux.h

```diff
@@ -51,6 +51,8 @@ class nixlBackendInitParams {
 
         bool              enableProgTh;
         nixlTime::us_t    pthrDelay;
+
+        unsigned int      numWorkers;
```

Comment:
is it needed for other backends? maybe it can be custom param?

Reply:
enableProgTh and pthrDelay are also UCX-specific, so I added numWorkers here for consistency. I don't have a strong opinion either way.

Reply:
Another approach could be to not expose this config to the user at all and instead use 1 worker when mt model is 'strict' or some predetermined value >1 when the model allows concurrent execution. WDYT?

Reply:
Moved it to custom param.